File: src/main/java/com/catan/catanbackend/model/dto/BankTradeDto.java
package com.catan.catanbackend.model.dto;

import com.catan.catanbackend.model.ResourceGroup;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class BankTradeDto {
    private Long sessionId;
    private String fromUser;
    private ResourceGroup offered;
    private ResourceGroup requested;
    private String portType;
    private int portRatio;
}

File: Assets/Scripts/TradingReasources/Models/BankTradeDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Assets.Scripts.TradingReasources.Models
{
    [Serializable]
    public class BankTradeDto
    {
        public long sessionId;
        public string fromUser;
        public ResourceGroup offered;
        public ResourceGroup requested;
        public string portType;
        public int portRatio;
    }
}
